cmd/earthly/app/run.go

```diff
@@ -384,3 +383,15 @@ func (app *EarthlyApp) printCrashLogs(ctx context.Context) {
 func errorWithPrefix(err string) string {
 	return fmt.Sprintf("Error: %s", err)
 }
+
+func getHintErr(err error, grpcError *status.Status) (*hint.Error, bool) {
+	if res := new(hint.Error); errors.As(err, &res) {
+		return res, true
+	}
+	if grpcError != nil {
+		if res, ok := hint.FromError(errors.New(grpcError.Message())); ok {
+			return res, ok
```

Comment:
nit: since ok can only ever be true here, I would just return `true` to reduce cognitive load on future readers.
```suggestion
			return res, true
```

Reply:
good point!
I realized I can just return from the function as is